ONSdigital/eq-author
# 184 date answer

### What is the context of this PR?

Adds the Date answer type with editable label (legend). Also, tweaked the hinting on the various icons so they look a bit better.

### How to review 

- Add a date answer
- Check label works with preview in runner
- Check I didn't break DateRange somehow
